### PR TITLE
w/series-2/numbers/dev: merge with prod

### PR DESCRIPTION
**Pull request by t-nick**

### CODE REVIEW

File: src/component.js [hunk 1]
```diff
@@ -53,49 +53,88 @@ class Block extends React.Component {
...
+                >
                   {this.getModifierValue('button') && (
                     <Button
-                      className={classNames(style.button, this.props.$theme.enabled('ui-btns-group__item'))}
+                      className={classNames(
+                        style.button,
+                        style['btns-group__item'],
+                        this.props.$theme.enabled('w-btns-group__item'),
+                      )}
                       linkClassName={style.link}
```
Comment (artemvasilkin) [inline src/component.js]: remove `linkClassName={style.link}`

File: src/component.js [hunk 2]
```diff
@@ -53,49 +53,88 @@ class Block extends React.Component {
...
                   )}
                   {this.getModifierValue('button_additional') && (
                     <Button
-                      className={classNames(style.button, this.props.$theme.enabled('ui-btns-group__item'))}
+                      className={classNames(
+                        style.button,
+                        style['btns-group__item'],
+                        this.props.$theme.enabled('w-btns-group__item'),
+                      )}
                       linkClassName={style.link}
```
Comment (artemvasilkin) [inline src/component.js]: remove `linkClassName={style.link}`

File: src/style.css [hunk 3]
```diff
@@ -63,11 +75,10 @@
 }
 
 .title {
-  margin-bottom: 15px;
   text-align: left;
```
Comment (artemvasilkin) [inline src/style.css]: remove `linkClassName={style.link}` у `.title` и `.subtitle`

File: src/component.js [hunk 4]
```diff
@@ -186,8 +225,8 @@ Block.modifierScheme = {
   button: {defaultValue: false, label: 'Button (link)', type: 'checkbox', sortOrder: 30},
   button_additional: {
     defaultValue: false,
-    label: 'Button additional',
-    type: 'hidden',
+    label: 'Button Additional (link)',
```
Comment (artemvasilkin) [inline src/component.js]: `Button Additional (link)` заменить на `Additional button (link)`

**Comment by t-nick** [comment]
Исправила

**Pull request merged**